.changeset/flat-flowers-remain.md

```diff
@@ -0,0 +1,10 @@
+---
+"@logto/core": minor
+"@logto/integration-tests": minor
+"@logto/schemas": minor
+---
+
+update hook schemas
+
+- add `name`, `events`, `signingKey` and `enabled` fields to the webhook schema.
+- deprecate `retries` field in the `HookConfig` schema and fix the webhook post request retry times to 3.
```

Comment:
i remember we talked about to use a summarized changeset instead of several

Reply:
Removed; I will add the changeset when all related PRs are ready.